packages/office-ui-fabric-react/src/components/ResizeGroup/ResizeGroup.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import {
+  css,
+  BaseComponent
+} from '../../Utilities';
+import { IResizeGroupProps } from './ResizeGroup.Props';
+import styles = require('./ResizeGroup.scss');
+
+export interface IResizeGroupState {
+  renderedData?: any;
+  measuredData: any;
+  shouldMeasure?: boolean;
+}
+
+export class ResizeGroup extends BaseComponent<IResizeGroupProps, IResizeGroupState> {
+
+  private _root: HTMLElement;
+  private _measured: HTMLElement;
+
+  constructor(props: IResizeGroupProps) {
+    super(props);
+    this.state = {
+      shouldMeasure: true,
+      renderedData: null,
+      measuredData: { ...this.props.data },
+    };
+  }
+
+  public componentDidMount() {
+    this._measureItems();
+    this._events.on(window, 'resize', this._onResize);
```

Comment:
do we also want to have an IResizeGroup interface with a method to trigger this manually?